config/pulse.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+use Laravel\Pulse\Http\Middleware\Authorize;
+use Laravel\Pulse\Pulse;
+use Laravel\Pulse\Recorders;
```

Comment:
Jak Ci się z tym pracuje? wyłapuje błędy? Zrobisz na instalację tego i podsumowanie osobne zadania? Chciałbym poznać Twoje zdanie o tym narzędziu

Reply:
Dopiero teraz przeczytałem ten komentarz, ciężko mi powiedzieć, bo zainstalowałem to chyba 2 dni temu, więc nie czuję się kompetentny żeby to recenzować. Wygląda fajnie, widać kilka rzeczy (zainstalowałem bo chciałem joby w tym podglądać). Wyłapuje jakieś błędy na pewno, ale czy w dużym projekcie będzie przydatne do debugowania to nie mam pojęcia. Do optymalizacji z pewnością, ale na ile będzie to czytelne i przydatne gdy zapytań będzie dużo więcej? Nie jestem w stanie powiedzieć...